monai/handlers/checkpoint_saver.py

```diff
@@ -37,6 +37,9 @@ class CheckpointSaver:
         name: identifier of logging.logger to use, if None, defaulting to ``engine.logger``.
         file_prefix: prefix for the filenames to which objects will be saved.
         save_final: whether to save checkpoint or session at final iteration or exception.
+            if checkpoints are to be saved when an exception is raised, put this handler before
```

Comment:
"if checkpoints" -> "If checkpoints"

Reply:
Almost all the doc-strings don't use the capital for the first char, so I just followed. What do you think?

Reply:
With "save_final: whether" the "whether" not being captilized makes sense as it's not really a strictly correct sentence  beginning with an identifier whose case shouldn't be changed. However any following sentences need to be correctly capitalized I think. If this is prevalent across a lot of the comments we can leave it off for now, we might need to do a whole codebase style review/cleanup at some point in the future.

Reply:
OK, I updated in the latest commit.
I agree with your proposal, we also need to define the expected doc-string format in contribution document first.

Thanks.